compose/ui/ui/src/uikitInstrumentedTest/kotlin/androidx/compose/ui/leaks/MemoryLeaksTest.kt

```diff
@@ -275,6 +275,171 @@ class MemoryLeaksTest {
             assertEquals(emptyList(), subviewsReferences.mapNotNull { it.get() })
         }
 
+    @Test
+    fun testComposeViewDisposal() = runRepeatingBlocking {
```

Comment:
```suggestion
    fun testComposeUIViewDisposal() = runRepeatingBlocking {
```

same for other tests?

Reply:
Done